vm_templates/common/node-js.yml

```diff
@@ -5,12 +5,14 @@ json:
       - 0.8.27
       - 0.10.18
       - 0.10.36
-      - 0.11.15
+      - 0.11.16
+      - 0.12.0
     aliases:
       "0.10": '0.1'
-      0.11.15: node-head
-      0.11.15: node-unstable
-    default: 0.10.36
+      "0.12": '0.1'
```

Comment:
This is incorrect. https://github.com/travis-ci/travis-cookbooks/blob/8500b48a5693066345609ea2529492c240444954/ci_environment/nodejs/recipes/multi.rb#L86-92 shows how the aliases are created.

Alias `0.1` for "0.10" is required because users make a mistake of writing

```yaml
node_js: 0.10
```

Without quotation marks, `0.10` is interpreted as a float, so that we get `0.1`. The alias `0.1` is needed to hide this common error.

No such fix is necessary for 0.12.